src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
+type Shape = 'triangle' | 'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
 
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  public shape: Shape = 'triangle';
+
+  isInvalidTriangle(): boolean {
```

Comment:
It is a cool method and I like it. Let's make it private, cause you are using it only internally